src/com/projectkorra/projectkorra/firebending/FireShield.java

```diff
@@ -240,20 +250,12 @@ public void setShieldDuration(final long duration) {
 		this.shieldDuration = duration;
 	}
 
-	public long getInterval() {
-		return this.interval;
-	}
-
-	public void setInterval(final long interval) {
-		this.interval = interval;
-	}
-
 	public double getRadius() {

```

Comment:
This method should be renamed to better reflect the specific variable it accesses. Something like `getShieldRadius` would be more fitting.